### PR TITLE
OTT-1564: [bug fix] avoid database calls in vast-unwrap module for hybrid+activation endpoint 

### PR DESCRIPTION
**Pull request by ashishshinde-pubm**

### CODE REVIEW

**Review by Pubmatic-Dhruv-Sonone — DISMISSED** [review]

File: modules/pubmatic/vastunwrap/entryhook.go [hunk 1]
```diff
@@ -40,14 +52,17 @@ func handleEntrypointHook(
 		}
 	} else {
 		endpoint := openwrap.GetEndpoint(payload.Request.URL.Path, source)
+		if _, ok := supportedEndpoints[endpoint]; !ok {
+			result.DebugMessages = append(result.DebugMessages, fmt.Sprintf("debug: %s endpoint does not support vast-unwrap feature", endpoint))
```
Comment (pm-nilesh-chate) [inline modules/pubmatic/vastunwrap/entryhook.go:56]: no need to added "debug: " tag, it is already in a separate debug section

**Review by pm-jaydeep-mohite — APPROVED** [review]
LGTM

**Review by pm-nilesh-chate — APPROVED** [review]
approved with minor comment

File: modules/pubmatic/vastunwrap/entryhook.go (hunk 1 above)
Comment (ashishshinde-pubm) [reply modules/pubmatic/vastunwrap/entryhook.go]: done

**Review by AvinashKapre — APPROVED** [review]

**Review by Pubmatic-Dhruv-Sonone — APPROVED** [review]

**Pull request merged**